k6/stress/stress.js

```diff
@@ -0,0 +1,44 @@
+import http from 'k6/http';
+import { check, group, sleep, fail } from 'k6';
+
+export let options = {
+    stages: [
+        // { duration: '2m', target: 200 },
+
+        { duration: '5m', target: 150 },
+        { duration: '5m', target: 200 },
+        { duration: '5m', target: 250 },
```

Comment:
계단형으로 그래프를 그리려면 이렇게 작성해야 해요~
```suggestion
        { duration: '20s', target: 150 },
        { duration: '5m', target: 150 },                
        { duration: '20s', target: 200 },        
        { duration: '5m', target: 200 },
        { duration: '20s', target: 250 },        
        { duration: '5m', target: 250 },
```